index.bs

```diff
@@ -357,10 +355,46 @@ execute the following steps:
 5.  [$Set the `Sec-CH-UA` header for a request$], given |r|.
 
 
-IDL {#idl} 
-===
+Interface {#interface} 
+=================
+
+<pre class="idl">
+[Exposed=Window]
+interface NavigatorUAData {
+  readonly attribute DOMString brand;
+  readonly attribute DOMString version;
+  readonly attribute DOMString platform;
+  readonly attribute DOMString architecture;
+  readonly attribute DOMString model;
+  readonly attribute boolean mobile;
+};
+interface mixin NavigatorUA {
+  [SecureContext] Promise&lt;NavigatorUAData&gt; getUserAgent();
+};
+Navigator includes NavigatorUA;
+
+</pre>
 
-ISSUE: Yoav should write all this.
+Processing model {#processing}
+--------------
+<dfn method for="NavigatorUA"><code>getUserAgent()</code></dfn> method *must*
```

Comment:
Nit: Elsewhere in the doc we're using caps (e.g. "MUST").

Reply:
ack